tools/test_deleted_renamed_referenced_files

```diff
@@ -19,7 +19,8 @@ for FILE in $FILES; do
         file_to_verify="$FILE" 
         target_paths='schedule/'
     fi
-    if MATCHED_SCHEDULE_FILES="$(grep --recursive --ignore-case --files-with-matches "${file_to_verify}\b" $target_paths)"
+    if [[ ! -z $file_to_verify ]]  && [[ MATCHED_SCHEDULE_FILES="$(grep --recursive --ignore-case \
```

Comment:
instead of negate the flag, `[[ -n STRING ]]` or `[[ STRING ]]` should work as well.

Reply:
Done